pkg/planner/cardinality/selectivity_test.go

```diff
@@ -2093,3 +2093,20 @@ func TestLastBucketEndValueHeuristic(t *testing.T) {
 		require.InDelta(t, 109.99, idxOtherCount.Est, 0.1, "Index other count should be approximately 109.99")
 	}
 }
+
+func TestUninitializedStats(t *testing.T) {
+	store, _ := testkit.CreateMockStoreAndDomain(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("set @@global.tidb_enable_auto_analyze = 'OFF';")
```

Comment:
I think this is unnecessary in the unit test because it doesn’t reach the analyze threshold.